### PR TITLE
[CLOUD-429] Fix runsvdir reboot issues with Upstart

### PR DESCRIPTION
**Pull request by ryancragun**
### Description
* Remove duplicate Ubuntu 14.04 kitchen platform
* Update gem dependencies
* Increase the Upstart kill timeout for runsvdir from 5 seconds to 30. Without
  this it was possible for Upstart to kill the runsvdir process before
  the children runsv processes had been terminated, leaving them orphaned
  to Upstart.
* Send `sv force-stop` to runsv processes before runsvdir is terminated.
  This ensures that processes that don't exit after 7 seconds are terminated.
* Transition the Upstart goal to 'stop' on reboot. This fixes a bug where
  Upstart would respawn runsvdir during a reboot, causing services to
  stop and start when the system is going down.

### Issues Resolved
Fixes upstart orphaning child runsv processes by killing timed out processes, increasing the overall kill timeout, and properly sets the upstart goal so services are not respawned during reboot.

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: resources/pg_database.rb [hunk 1]
```diff
@@ -1,31 +1,58 @@
...
-required: false
-
-attribute :host,
-kind_of: String,
-required: false
+property :database, String, name_property: true
+property :owner, String, required: false
+property :template, String, default: 'template0'
+property :encoding, String, default: 'UTF-8'
+property :username, String, required: false
```
Comment (tas50) [inline resources/pg_database.rb]: required false isn't necessary. False is the default

File: Gemfile.lock [hunk 2]
```diff
@@ -1,318 +1,251 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (4.2.4)
```
Comment (tas50) [inline Gemfile.lock]: Why are we shipping a lock file?

File: resources/pg_database.rb [hunk 3]
```diff
@@ -1,31 +1,58 @@
-actions :create
-
 default_action :create
```
Comment (tas50) [inline resources/pg_database.rb]: No need for default action

File: Gemfile.lock (hunk 2 above)
Comment (ryancragun) [inline Gemfile.lock]: I can't really answer that. It was already there so I assumed it had a purpose.

**Pull request merged**